runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7107,6 +7107,19 @@ done:;
 		return rc;
 	}
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	VMINLINE bool
+	acmp(j9object_t lhs, j9object_t rhs) 
+	{
+		if(lhs != rhs) return false;
```

Comment:
please dont use inline if statements. See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md

Reply:
Also, only have a single return statement in a function, use a temp variable if you have to.